modules/packages/npm/metadata.go

```diff
@@ -15,6 +15,7 @@ type Metadata struct {
 	Author                  string            `json:"author,omitempty"`
 	License                 string            `json:"license,omitempty"`
 	ProjectURL              string            `json:"project_url,omitempty"`
+	Repository              Repository        `json:"repository,omitempty"`
```

Comment:
I would move this (as a simple url string) into the `Package` class as there is no use for this value after the upload.